src/CalChartFrame.cpp

```diff
@@ -964,7 +965,10 @@ void CalChartFrame::OnCmd_ShowBackgroundImages(wxCommandEvent& event)
 void CalChartFrame::OnCmd_GhostOption(wxCommandEvent& event)
 {
     auto selection = static_cast<wxChoice*>(FindWindow(CALCHART__GhostControls))->GetSelection();
-    auto which_option = (event.GetId() == CALCHART__GhostControls) ? selection : (event.GetId() == CALCHART__GhostOff) ? 0 : (event.GetId() == CALCHART__GhostNextSheet) ? 1 : (event.GetId() == CALCHART__GhostPreviousSheet) ? 2 : 3;
+    auto which_option = (event.GetId() == CALCHART__GhostControls) ? selection : (event.GetId() == CALCHART__GhostOff) ? 0
```

Comment:
Woah this is like a switch-case, but its an expression instead of a statement. Very interesting